make_queue/forms.py

```diff
@@ -153,3 +154,27 @@ class FreeSlotForm(forms.Form):
         initial=1, label=_("Machine type"))
     hours = IntegerField(min_value=0, initial=1, label=_("Duration in hours"))
     minutes = IntegerField(min_value=0, max_value=59, initial=0, label=_("Duration in minutes"))
+
+
+class BaseMachineForm(forms.ModelForm):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.fields["status"] = forms.ChoiceField(choices=(
+            ("F", _("Available")),
+            ("O", _("Out of order")),
+            ("M", _("Maintenance")),
+        ))
+
+    class Meta:
+        model = Machine
+        fields = "__all__"
+        widgets = {
+            "status": SemanticChoiceInput(),
```

Comment:
This does not seem to work for me. I only get the default chrome select, without the Semantic UI classes. It does, however, work if I set the widget in the ChoiceField constructor. I do not think that `widget` in the `Meta` class generally works for self defined fields.